src/cdk/v2/destinations/smartly/data/trackMapping.json

```diff
@@ -0,0 +1,69 @@
+[
+  {
+    "sourceKeys": "event",
+    "required": true,
+    "destKey": "event"
+  },
+  {
+    "destKey": "value",
+    "sourceKeys": [
+      "properties.total",
+      "properties.value",
+      "properties.revenue",
+      {
+        "operation": "multiplication",
+        "args": [
+          {
+            "sourceKeys": "properties.price"
+          },
+          {
+            "sourceKeys": "properties.quantity",
+            "default": 1
+          }
+        ]
+      }
+    ],
+    "metadata": {
+      "type": "toNumber"
+    },
+    "required": false
+  },
+  {
+    "sourceKeys": ["properties.conversions", "properties.products.length"],
+    "required": false,
+    "metadata": {
+      "defaultValue": "1"
```

Comment:
why default value is 1? can you explain this?

Reply:
Because they have mentioned in the doc that :
conversions: how many conversion events happened (optional, defaults to 1 if not provided)
https://support.smartly.io/hc/en-us/articles/4406049685788-S2S-integration-API-description#01H8HBXZF6WSKSYBW1C6NY8A88

Reply:
please add this reference to the code itself